src/plugins/ovp/components/jwplayer-media-fields/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Card, CardBody, CardHeader } from 'reactstrap';
```

Comment:
@rsumilang field names are invalid.  can remove placeholders as well.  also this component can just be a component within video-screen with a name matching the mixin that provides the fields (like notification types, app types, etc